Update tutorial links after big split

The link check is failing in master, causing current PRs to fail:  

```
+py.test --check-links -k .md .
============================= test session starts ==============================
platform linux -- Python 3.6.0, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /home/travis/build/jupyterlab/jupyterlab, inifile: pytest.ini
plugins: check-links-0.1.0
collected 113 items 
CONTRIBUTING.md ..........
README.md .....................
RELEASE.md ..
design/about.md .
tutorial/adding_content.md .FF.
tutorial/documents.md ..........
tutorial/examples.md ..
tutorial/labextensions.md ..
tutorial/notebook.md ..................
tutorial/patterns.md ..
tutorial/plugins.md F.FFFFFFFFFF.
tutorial/repo.md .
tutorial/virtualdom.md ..
=================================== FAILURES ===================================
 /home/travis/build/jupyterlab/jupyterlab/tutorial/adding_content.md: https://github.com/jupyterlab/jupyterlab/blob/master/typings/ansi_up/ansi_up.d.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/typings/ansi_up/ansi_up.d.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/adding_content.md: https://github.com/jupyterlab/jupyterlab/blob/master/typings/xterm/xterm.d.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/typings/xterm/xterm.d.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/tree/master/examples/lab 
https://github.com/jupyterlab/jupyterlab/tree/master/examples/lab: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/terminal-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/terminal-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/shortcuts-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/shortcuts-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/imagewidget-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/imagewidget-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/help-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/help-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/filebrowser-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/filebrowser-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/editorwidget-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/editorwidget-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/console-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/console-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/services-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/services-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/rendermime-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/rendermime-extension/index.ts: HTTP Error 404: Not Found
 /home/travis/build/jupyterlab/jupyterlab/tutorial/plugins.md: https://github.com/jupyterlab/jupyterlab/blob/master/src/docregistry-extension/index.ts 
https://github.com/jupyterlab/jupyterlab/blob/master/src/docregistry-extension/index.ts: HTTP Error 404: Not Found
============================= 25 tests deselected ==============================
============= 13 failed, 75 passed, 25 deselected in 8.35 seconds ==============
```

--- a/tutorial/adding_content.md
+++ b/tutorial/adding_content.md
@@ -8,7 +8,7 @@ As an example: Add a leaflet viewer plugin for geoJSON files.
 files: `npm install -g tsd`.
 - Try to install the typings files from the top level source directory:
 `tsd install leaflet`. (success!)
-- If there are no typings, we must create our own.  An example typings file that exports functions is [ansi_up](https://github.com/jupyterlab/jupyterlab/blob/master/typings/ansi_up/ansi_up.d.ts).  An example with a class is [xterm](https://github.com/jupyterlab/jupyterlab/blob/master/typings/xterm/xterm.d.ts).
+- If there are no typings, we must create our own.  An example typings file that exports functions is [ansi_up](https://github.com/jupyterlab/jupyterlab/blob/master/packages/rendermime/typings/ansi_up/ansi_up.d.ts).  An example with a class is [xterm](https://github.com/jupyterlab/jupyterlab/blob/master/packages/terminal/typings/xterm/xterm.d.ts).
 - Add a reference to the new library in `src/typings.d.ts`.
 - Create a folder in `src` for the plugin.
 - Add  `index.ts` and `plugin.ts` files.

--- a/tutorial/plugins.md
+++ b/tutorial/plugins.md
@@ -12,7 +12,7 @@ A JupyterLab application is comprised of:
 - A core Application object
 - Plugins
 
-A full example of an application is contained [here](https://github.com/jupyterlab/jupyterlab/tree/master/examples/lab).
+A full example of an application is contained [here](https://github.com/jupyterlab/jupyterlab/tree/master/examples/app).
 Looking at the `index.js` file, you can see the extensions
 used in the tutorial example.
 
@@ -27,17 +27,17 @@ a typed value to the plugin's `activate()` method.
 exporting a plugin object or array of plugin objects as the default export.
 
 The default plugins in the JupyterLab application include:
-- [Terminal](https://github.com/jupyterlab/jupyterlab/blob/master/src/terminal-extension/index.ts) - Adds the ability to create command prompt terminals.
-- [Shortcuts](https://github.com/jupyterlab/jupyterlab/blob/master/src/shortcuts-extension/index.ts) - Sets the default set of shortcuts for the application.
-- [Images](https://github.com/jupyterlab/jupyterlab/blob/master/src/imagewidget-extension/index.ts) - Adds a widget factory for displaying image files.
-- [Help](https://github.com/jupyterlab/jupyterlab/blob/master/src/help-extension/index.ts) - Adds a side bar widget for displaying external documentation.
-- [File Browser](https://github.com/jupyterlab/jupyterlab/blob/master/src/filebrowser-extension/index.ts) - Creates the file browser and the document manager and the file browser to the side bar.
-- [Editor](https://github.com/jupyterlab/jupyterlab/blob/master/src/editorwidget-extension/index.ts) - Add a widget factory for displaying editable source files.
-- [Console](https://github.com/jupyterlab/jupyterlab/blob/master/src/console-extension/index.ts) - Adds the ability to launch Jupyter Console instances for
+- [Terminal](https://github.com/jupyterlab/jupyterlab/blob/master/packages/terminal-extension/src/index.ts) - Adds the ability to create command prompt terminals.
+- [Shortcuts](https://github.com/jupyterlab/jupyterlab/blob/master/packages/shortcuts-extension/src/index.ts) - Sets the default set of shortcuts for the application.
+- [Images](https://github.com/jupyterlab/jupyterlab/blob/master/packages/imagewidget-extension/src/index.ts) - Adds a widget factory for displaying image files.
+- [Help](https://github.com/jupyterlab/jupyterlab/blob/master/packages/help-extension/src/index.ts) - Adds a side bar widget for displaying external documentation.
+- [File Browser](https://github.com/jupyterlab/jupyterlab/blob/master/packages/filebrowser-extension/src/index.ts) - Creates the file browser and the document manager and the file browser to the side bar.
+- [Editor](https://github.com/jupyterlab/jupyterlab/blob/master/packages/editorwidget-extension/src/index.ts) - Add a widget factory for displaying editable source files.
+- [Console](https://github.com/jupyterlab/jupyterlab/blob/master/packages/console-extension/src/index.ts) - Adds the ability to launch Jupyter Console instances for
 interactive kernel console sessions.
-- [Services](https://github.com/jupyterlab/jupyterlab/blob/master/src/services-extension/index.ts) - An application-specific interface to `@jupyterlab/services`.
-- [RenderMime](https://github.com/jupyterlab/jupyterlab/blob/master/src/rendermime-extension/index.ts) - The registry for adding kernel `display_data` renderers.
-- [Document Registry](https://github.com/jupyterlab/jupyterlab/blob/master/src/docregistry-extension/index.ts) - Used to add functionality around widgets backed by files.
+- [Services](https://github.com/jupyterlab/jupyterlab/blob/master/packages/services-extension/src/index.ts) - An application-specific interface to `@jupyterlab/services`.
+- [RenderMime](https://github.com/jupyterlab/jupyterlab/blob/master/packages/rendermime-extension/src/index.ts) - The registry for adding kernel `display_data` renderers.
+- [Document Registry](https://github.com/jupyterlab/jupyterlab/blob/master/packages/docregistry-extension/src/index.ts) - Used to add functionality around widgets backed by files.
 
 ## Application Object
 The JupyterLab Application object is given to each plugin in

